packages/cli/src/ui/commands/promptCommands.ts

```diff
@@ -0,0 +1,183 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  SlashCommand,
+  SlashCommandActionReturn,
+  CommandContext,
+  CommandKind,
+} from './types.js';
+import {
+  Config,
+  getMCPServerPrompts,
+  getErrorMessage,
+  invokeMcpPrompt,
+  connectToMcpServer,
+} from '@google/gemini-cli-core';
+
+export function createPromptCommands(config: Config | null): SlashCommand[] {
+  const promptCommands: SlashCommand[] = [];
+  if (config) {
```

Comment:
nit: How do you feel about early. returning here to reduce nesting? I.e. `if (!config) { return promptCommands; }`